llvm/lib/CodeGen/MachinePipeliner.cpp

```diff
@@ -1327,47 +1327,6 @@ class HighRegisterPressureDetector {
   void computePressureSetLimit(const RegisterClassInfo &RCI) {
     for (unsigned PSet = 0; PSet < PSetNum; PSet++)
       PressureSetLimit[PSet] = TRI->getRegPressureSetLimit(MF, PSet);
-
```

Comment:
Hi @kasuga-fj! What do you think about this part? I just removed the code below as it seems to be unnecessary now. Related patch: https://github.com/llvm/llvm-project/pull/74807

Reply:
And I think just removing `fixed` registers is not enough here.

Reply:
I dared to replace it from `RegisterClassInfo::getRegPressureSetLimit` before (in #87312). This is because there were duplicate registers between what `RegisterClassInfo::getRegPressureSetLimit` removes and the `fixed` registers mentioned here. If `RegisterClassInfo::getRegPressureSetLimit` now takes care of both, I don't think it's a problem to remove this part as you did.